sockeye/transformer.py

```diff
@@ -69,6 +69,7 @@ class TransformerEncoderBlock:
     def __init__(self,
                  config: TransformerConfig,
                  prefix: str) -> None:
+        self.dtype = config.dtype
```

Comment:
why is this needed? couldn't we use the (potentially overriden) `self.config.dtype`?

Reply:
you are right, missed that config is a member already.